install_config/install/advanced_install.adoc

```diff
@@ -663,6 +663,7 @@ If you are using an image registry other than the default at
 ----
 oreg_url=example.com/openshift3/ose-${component}:${version}
 openshift_examples_modify_imagestreams=true
+openshift_hosted_manage_registry=false 
```

Comment:
openshift_hosted_manage_registry and openshift_hosted_manage_router determine installer would deploy registry/router or not, this option has nothing with "Configuring a Registry Location".  If user are using an image registry other than the default at registry.access.redhat.com, installer would deploy router/registry with 'openshift3/ose-${component}:${version}' image by default into this cluster, where to pull image depends on openshift_docker_additional_registries setting, if user set openshift_docker_additional_registries=example.com in the inventory host file, everything is working well. So openshift_hosted_manage_registry option should be replaced by openshift_docker_additional_registries. Of course, if user wan to deploy registry/router later after installation is completed, openshift_hosted_manage_registry and openshift_hosted_manage_router setting would be meaningful.

Reply:
I think I understand. It is so strange that `openshift_hosted_manage_registry=false` is explained in the **Configuring a Registry Location** section. Replacing it with `openshift_docker_additional_registries` might be one option, but it would be better to simply delete the `openshift_hosted_manage_registry=false` and its explanation from **Configuring a Registry Location** section.

Reply:
Hmm... without `openshift_docker_additional_registries=example.com`, current setting below makes a failure of deploying registry image? If so, `openshift_docker_additional_registries=example.com` would be better to be added in the doc as @jiajliu mentioned (though it is different topic from this PR).

~~~
 oreg_url=example.com/openshift3/ose-${component}:${version}
 openshift_examples_modify_imagestreams=true
~~~

Reply:
It would be perfect if openshift_docker_additional_registries replace this line, because if user are using a customized registry, but do not openshift_docker_additional_registries, installer would pull registry/router images from product registry. Maybe that is not the behavior which user has not expected.
Of course, if it is also acceptable to simply delete the openshift_hosted_manage_registry line, at latest, we did not make it worse.

Reply:
Ummmm...If `openshift_docker_additional_registries=example.com` is set, the configuration becomes:

~~~
add_registry=--add-registry registry.access.redhat.com --add-registry example.com
~~~

So, as far as `registry.access.redhat.com` is  accessible, I think users cannot change the registry to pull the image of registry and still pull it from `registry.access.redhat.com` (because it is first registry in the line). But, it may be natural that we assume that `registry.access.redhat.com` is blocked or disconnected, when users want to change the location.  But it is our "assumption"... 

Anyway, I think that it is alright to replace the line.

Reply:
Interesting, in my testing, openshift_docker_additional_registries=example.com, the docker config file would have such lines which is different with yours:
ADD_REGISTRY='--add-registry example.com --add-registry registry.access.redhat.com'

Reply:
Umm, you are correct. `oo_prepend_strings_in_list` should add it as `prepend`. Probably, it is my bad. Let's replace it with `openshift_docker_additional_registries`.